translations/invoice-renderer.en.xlf

```diff
@@ -20,20 +20,20 @@
         <target>HTML (for printing via Browser)</target>
       </trans-unit>
       <trans-unit id="4NNx9U." resname="Docx">
-        <source>Docx</source>
-        <target>Microsoft Word (Docx)</target>
+        <source>Microsoft Word (DOCX)</source>
```

Comment:
VERIFY! Probably reject...

Reply:
It is either DOCX or .docx https://docs.fileformat.com/word-processing/docx/